.github/workflows/fleximod_test.yaml

```diff
@@ -0,0 +1,31 @@
+name: git-fleximod test
+#
+# Test git-fleximod update and cleanliness
+# Based closely on workflow from CESM repo
+#
+on: [push, pull_request]
+
+jobs:
+  fleximod-test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # oldest supported and latest supported
+        python-version: ["3.7", "3.x"]
```

Comment:
Cool, so 3.x get's translated into the very latest python 3. version? That's really cool here.

Reply:
Yep! That's something I took straight from the CESM implementation of this workflow. Most of the workflow is like that, actually.